db/seeds.d/61-foreman_tasks_bookmarks.rb

```diff
@@ -0,0 +1,16 @@
+class AddForemanTasksBookmarks < ActiveRecord::Migration
+  def up
+    Bookmark.where(:name => "running",
+                   :query => "state = running",
+                   :controller => "foreman_tasks_tasks",
+                   :public => true).first_or_create
+    Bookmark.where(:name => "failed",
+                   :query => "state = paused || result = error || result = warning",
+                   :controller => "foreman_tasks_tasks",
+                   :public => true).first_or_create
+  end
+
+  def down
+    Bookmark.where(:controller => "foreman_tasks_tasks").map(&:destroy)
```

Comment:
Also, it seems `and` and `or` operations are somehow preferred over `&&` and `||`